### DISCUSSION TITLE
Delay in the scrutiny of the Student Developer Pack inspite of approval of my request

### DISCUSSION THREAD

**Discussion by Aman-Dinesh-Mishra** [community/community#143487] [GitHub Education]
### Hi everyone,

**It has been 3 days since my request has been approved for students developer benefits but I am unable to access the benefits . How long will it take for me to access the benefits od Github Student Developer Pack?**

**Comment by davevad93** [comment]
Hi @Aman-Dinesh-Mishra ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Aman-Dinesh-Mishra was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Aman-Dinesh-Mishra** [reply]
Thanks for the guidance

**Reply by davevad93** [reply]
You're welcome @Aman-Dinesh-Mishra 👍
